sklearn/metrics/_ranking.py

```diff
@@ -334,6 +336,12 @@ def det_curve(y_true, y_score, pos_label=None, sample_weight=None):
     roc_curve : Compute Receiver operating characteristic (ROC) curve.
     precision_recall_curve : Compute precision-recall curve.
 
+    Notes
+    -----
+    .. versionchanged:: 1.6
+      An arbritrary threshold at infinity is added to represent a classifier
+      that always predicts the negative class, i.e. `fpr=0` and `fnr=1`.
```

Comment:
I would rather move this below the `.. versionadded:: 0.24` after the main paragraph of the docstring instead of adding a new "Notes" section to this docstring.